git/homework.py

```diff
@@ -97,14 +109,14 @@ def is_word_in_text(word: str, text: str) -> bool:
         >>> False
 
     """
-    pass
+    return word in text.split()
```

Comment:
you do not need `.split()`. `split` converts string to list, operator `in` searches substring in str value too

Reply:
Done!